### PR TITLE
Make it easier to sign for non-tech people

### PR DESCRIPTION
**Pull request by Lesik**
Maybe this prevents all those PRs with files in the wrong place, etc.

### CODE REVIEW

**Comment by imachug** [comment]
LGTM! What about using `https://github.com/johncamilton` as link example, because many people use GitHub and don't have a personal site?

**Comment by borntohonk** [comment]
> Maybe this prevents all those PRs with files in the wrong place, etc.

Why would non-tech people care about the fsf?

**Comment by ninapaley** [comment]
I'm not 100% non-tech, but low-tech, and although I now have a GitHub account to upload SVG files directly to my project's repository, I am still not sure how to sign this letter. I absolutely would have wanted to sign even before I had this account, and I know many others who care but don't know how to sign.

**Comment by what-am-i-doing** [comment]
> 
> 
> > Maybe this prevents all those PRs with files in the wrong place, etc.
> 
> Why would non-tech people care about the fsf?

I'm an industrial electrician. I use GNU/Linux on my computer and I for one appreciate the enormous contributions the FSF has made in delivering such an excellent operating system. And I also recognize that Mr. Stallman has been a huge part of the FSF for most of its existence. He is now being attacked for, what... being autistic, and making people uncomfortable because of the associated brashness, social missteps, and focus on logic over diplomacy? The one marginalized group that it's apparently ok to hate. That doesn't sit right with me.

File: README.md [hunk 1]
```diff
@@ -11,21 +11,23 @@ Without the `<>`.
 
 Example:
 ```yaml
-name: Example name
-link: https://example.com/
+name: John Camilton
```
Comment (TheChymera) [inline README.md:14]: maybe not use an actual example?

**Comment by Lesik** [comment]
> Why would non-tech people care about the fsf?

For one, free software is not only a technical but also a philosophical, ethical and social matter. Think journalists and activists who depend on free software and the essential freedoms to do their work, but are usually not tech people. Or economically disadvantaged people, who can't afford expensive licenses but want to do private computing without paying with their user data to SaaS companies.

There is clearly an overlap between people who care about free software and people who've never used Github before, just look at the PR and issue pages of this repository for proof :)

File: README.md (hunk 1 above)
Comment (Lesik) [reply README.md:14]: I had the opinion that `Example name` might not be so clear as a real name, that's why I changed it. But that's up for discussion.

**Comment by daveout** [comment]
I support this.

**Pull request merged**